examples/flutter_gallery/lib/gallery/app.dart

```diff
@@ -138,6 +138,33 @@ class GalleryAppState extends State<GalleryApp> {
       );
     }
 
+    final Map<String, WidgetBuilder> _kRoutes = new Map<String,
+      WidgetBuilder>.fromIterable(
+      // For a different example of how to set up an application routing table,
```

Comment:
I think it would be better (now) to point devs at https://docs.flutter.io/flutter/widgets/Navigator-class.html. There's a much simpler example there.

Reply:
Done.